src/app.js

```diff
@@ -1 +1,44 @@
+/* eslint-disable no-shadow */
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = (source, destination) => {
+  if (!source || !destination) {
+    throw new Error('Source and destination must be provided');
+  }
+
+  if (source === destination) {
+    throw new Error('Source and destination paths are the same');
```

Comment:
The error message could be more descriptive. Consider specifying that the source and destination paths should be different.